src/core/types/addons.js

```diff
@@ -1,8 +1,5 @@
 /* @flow */
-import { validAddonTypes } from 'core/constants';
-
-// You have to just laugh at this line!
-type AddonTypeType = validAddonTypes;
```

Comment:
Did this work? If so, could we just move this silly line into `core/constants` to remove the duplication? Or is this one of the things the new Flow doesn't like?